lambda_code/src/models/errors.py

```diff
@@ -61,20 +59,15 @@ class CoarseValidationError(ValidationError):
     """Pre validation error"""
     message: str
 
-    def to_operation_outcome(self) -> OperationOutcome:
+    def to_operation_outcome(self) -> dict:
         return create_operation_outcome(
             resource_id=str(uuid.uuid4()), severity=Severity.error, code=Code.invariant, diagnostics=self.message)
 
 
-def create_operation_outcome(resource_id: str, severity: Severity, code: Code, diagnostics: str) -> OperationOutcome:
-    model = {
+def create_operation_outcome(resource_id: str, severity: Severity, code: Code, diagnostics: str) -> dict:
```

Comment:
Can we not use OperationOutcome at all any more? If not could you document why in here.